domains/acelino.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "AcelinoMargotti",
+    "email": "acelinobdeom@gmail.com"
+  },
+  "records": {
+    "CNAME": "acelinomargotti.github.io/"
```

CNAME doesnt allow any trailing slashes (/)
```suggestion
    "CNAME": "acelinomargotti.github.io"
```